src/Sparrow.Server/Sodium.cs

```diff
@@ -270,5 +272,31 @@ static Sodium()
 
         [DllImport(LIBSODIUM)]
         public static extern int sodium_mlock(byte* addr, UIntPtr len);
+
+        private static long _lockedBytes;
+
+        public static Size LockedMemory => new Size(_lockedBytes, SizeUnit.Bytes);
```

Comment:
This is only used in the debug endpoint. Better to expose it directly and use the `new Size()` there for better code symmetry

Reply:
done